### DISCUSSION TITLE
ACERCA DE PROYECTO

### DISCUSSION THREAD

**Discussion by paulajimeneza** [rramosp/ai4eng.v1#90] [00 - General]
Buenas tardes profe, quisiera saber si este proyecto me sirve:
https://www.kaggle.com/c/ventilator-pressure-prediction/code
Gracias!

**Comment by rramosp** [comment]
hola Paula,

sí, ese está bien ... tienes que considerarlo como series temporales,
usando el campo time_step como referencia ...  es decir, tendrás que montar
un dataset para predecir la presión del ventilador dado un conjunto de
observaciones pasadas.

los laboratorios 03.02 y 05.01 te ayudarán en eso

saludos
raúl

El vie, 17 dic 2021 a las 15:10, paulajimeneza ***@***.***>)
escribió:

> Buenas tardes profe, quisiera saber si este proyecto me sirve:
> https://www.kaggle.com/c/ventilator-pressure-prediction/code
> Gracias!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/90>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIN7HHPTLQ32STMHCQ3UROKLPANCNFSM5KJT5CPA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
